### PR TITLE
Add reports and leaderboard, and remove PHP

### PR DESCRIPTION
**Pull request by sschiessl-bcp**
@mattbeckman

How do I link the reports properly?

@ryanRfox 

Please approve the content of the reports

### CODE REVIEW

File: src/view-models/leaderboard.js [hunk 1]
```diff
@@ -4,7 +4,20 @@ import {Router} from 'aurelia-router';
 @inject(Router)
 export class Leaderboard {
 
-	hackers = [];
+	hackers = [
+		{
+			name: "Anonymous",
+			github: "htd.rmg.bounty42",
+			bugsReported: 3,
+			totalReward: 17333
```
Comment (ryanRfox) [inline src/view-models/leaderboard.js]: Please add " bitUSD" to this reward and on line 18 as well. Future remittance will (likely) be in bitCNY, so prefer not to add the label into the column heading.

**Comment by mattbeckman** [comment]
Linking to the full path is fine if you stick with PDFs (e.g. /src/assets/reports/Report123.pdf).

**Review by ryanRfox — CHANGES_REQUESTED** [review]
I am happy with the PDF content. I would prefer MD to enable easier collaboration during drafting and standardized in-browser rendering. 

I have one request below related to labeling the rewards.

**Comment by sschiessl-bcp** [comment]
> I am happy with the PDF content. I would prefer MD to enable easier collaboration during drafting and standardized in-browser rendering.

I will look into this for next report and eventually change the old ones.

**Comment by sschiessl-bcp** [comment]
@mattbeckman please merge

**Review by ryanRfox — APPROVED** [review]

**Comment by ryanRfox** [comment]
Yes. I made my Review approval. (FYI - I do not have write access; which is just fine).

**Pull request merged**